src/plugins/console/server/lib/elasticsearch_proxy_config.test.js

```diff
@@ -92,11 +91,9 @@ describe('plugins/console', function () {
           },
         };
 
-        expect(agent.options.checkServerIdentity)
-          .withArgs('right.com', cert)
-          .to.not.throwException();
+        expect(() => agent.options.checkServerIdentity('right.com', cert)).not.toThrow();
         const result = agent.options.checkServerIdentity('right.com', cert);
-        expect(result).to.be(undefined);
+        expect(result).toBe(undefined);
```

Comment:
Nit: this can be `toBeUndefined()`

Reply:
These changes are mostly done with codemods, so going to leave it since it doesn't make any functional difference and keeps with the existing usage. We can always follow-up with a PR across the repo afterwards.